lib/loader/index.js

```diff
@@ -235,6 +332,15 @@ module.exports = {
 
       try {
         injectionNode.appendChild(link);
+
```

Comment:
If `appendChild` and the resulting script inject is synchronous, this is fine, but if it isn't you may consider putting 336-343 in the `onload`.